src/Processors/Formats/Impl/PostgreSQLOutputFormat.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <Processors/Formats/IRowOutputFormat.h>
+#include <Core/Block.h>
+
+#include <Core/PostgreSQLProtocol.h>
+#include <Formats/FormatSettings.h>
+
+namespace DB
+{
+
+class PostgreSQLOutputFormat final : public IOutputFormat
```

Comment:
Could you please add a comment here with a link to the description of this format?

Reply:
Ok, I will add a comment in code. As for know you can read about it in https://www.postgresql.org/docs/current/protocol-flow.html#id-1.10.5.7.4